### PR TITLE
Devnet interactions

### PR DESCRIPTION
**Pull request by psorinionut**

### CODE REVIEW

File: metabonding/interaction/devnet.snippets.sh [hunk 1]
```diff
@@ -0,0 +1,190 @@
+WALLET_PEM="/home/elrond/elrondsdk/erdpy-venv/lib/python3.8/site-packages/erdpy/testnet/wallets/users/mike.pem"
```
Comment (andreibancioiu) [inline metabonding/interaction/devnet.snippets.sh:1]: Note to myself: there should be an easier way to reference wallets in the IDE.

File: metabonding/interaction/devnet.snippets.sh [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+PROXY="https://devnet-gateway.elrond.com"
+CHAIN_ID="D"
+
+METABONDING_WASM_PATH="/home/elrond/Github/sc-metabonding-rs/metabonding/output/metabonding.wasm"
```
Comment (andreibancioiu) [inline metabonding/interaction/devnet.snippets.sh:5]: I think here we can use a relative path.

File: metabonding/interaction/devnet.snippets.sh [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+
+SIGNER_PUB_KEY=0x52773868c13654355cca16adb389b09201fabf5d9d4b795ebbdae5b361b46f20
+deployMetabondingSC() {
+    erdpy --verbose contract deploy --recall-nonce \
+        --bytecode=${METABONDING_WASM_PATH} \
+        --pem=${WALLET_PEM} \
+        --gas-limit=200000000 \
+        --proxy=${PROXY} --chain=${CHAIN_ID} \
+        --arguments ${SIGNER_PUB_KEY} \
+        --send || return
```
Comment (andreibancioiu) [inline metabonding/interaction/devnet.snippets.sh:18]: I think (not sure) that we should use `|| return 1`.

**Review by andreibancioiu — APPROVED** [review]
:rocket:

**Review by catalinnnn — APPROVED** [review]

**Pull request merged**